### ISSUE TITLE
[Broken Website] xkcd What If

### ISSUE THREAD

**Issue by keli5** [darkreader/darkreader#9301]
# Broken Website Report

## Website Address
https://what-if.xkcd.com/

## Steps To Reproduce
Go to https://what-if.xkcd.com/ and try to activate Dark Reader.

## Problematic Observation
Even though the site is light-colored and has no inbuilt dark mode to enable, Dark Reader cannot be activated.

## System Information

- OS: Xubuntu 21.10
- Browser: Firefox 102.0 64-bit
- Dark Reader Version: 4.9.52

**Comment by Gusted** [comment]
Hi,

Do you have by any chance xkcd.com disabled?

**Comment by keli5** [comment]
xkcd.com was not disabled, but for some reason, after reinstalling the extension, it started working again.
